Source/Routing and Faulting/Custom Device Routing and Faulting.xml

```diff
@@ -60,10 +60,10 @@
         <loc>Routing and Faulting</loc>
       </Name>
       <GUID>4e591626-626d-4903-9a6e-1f2a7b879f4d</GUID>
-      <Glyph>
-        <Type>To Application Data Dir</Type>
-        <Path>System Explorer\Glyphs\default fpga category.png</Path>
-      </Glyph>
+			<Glyph>
+				<Type>To Common Doc Dir</Type>
+				<Path>Custom Devices\Routing and Faulting\Windows\Glyphs\routing and faulting page.png</Path>
+			</Glyph>
```

Comment:
This file is a mix of tabs and spaces, we should be consistent.

Reply:
Yep, fixed.